pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -474,6 +507,11 @@ func (k *Bootstrapper) restartCluster(cfg config.ClusterConfig) error {
 		return nil
 	}
 
+	if err := kverify.NodePressure(client); err != nil {
```

Comment:
Calling this does not make sense here, as there may not be a working apiserver to talk to.

Reply:
I can make a defer, so if they specify normal wait, it will run after default wait compoents

Reply:
done

Reply:
Please move this to be called some time after `WaitForAPIServerProcess`